lib/ansible/modules/cloud/cloudstack/cs_nic.py

```diff
@@ -205,14 +214,14 @@ def get_secondary_ip(self):
 
     def present_nic(self):
         nic = self.get_nic()
-        if not self.get_secondary_ip():
+        if self.vm_guest_ip and not self.get_secondary_ip():
```

Comment:
Yep it might break things and that's why I needed your opinion. The problem with the module I found is that if `vm_guest_ip` isn't given, it still automatically means that we must add a secondary IP, implying that it will come from a DHCP/automatically from cloudstack. The module needs another switch, or a 3-states one for the IP: A - I don't want an IP, B - I want an IP automatically, C - I want this specific IP assigned to the nic.
Should I add another parameter, for example `add_ip` with default to `True` to not break compatibility with current configuration, to fetch and assign a secondary IP to the nic on the given network?